nomos-da/network/core/Cargo.toml

```diff
@@ -5,6 +5,7 @@ name    = "nomos-da-network-core"
 version = "0.1.0"
 
 [dependencies]
+cached                   = { version = "0.55.1", default-features = false }
```

Comment:
Likewise

Reply:
In fact, @ntn-x2 , what is the heuristic to choose workspace or local deps now? :thinking: